### PR TITLE
Text analytics update error model

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

File: sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java [hunk 1]
```diff
@@ -14,11 +14,11 @@
 import com.azure.cs.textanalytics.implementation.TextAnalyticsClientImpl;
 import com.azure.cs.textanalytics.implementation.models.DocumentLanguage;
 import com.azure.cs.textanalytics.implementation.models.LanguageBatchInput;
-import com.azure.cs.textanalytics.implementation.models.LanguageResult;
+import com.azure.cs.textanalytics.implementation.models.LanguageResult;	import com.azure.cs.textanalytics.models.Error;
```
Comment (mssfang) [inline sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java]: should be separated lines

File: sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java [hunk 2]
```diff
@@ -275,8 +275,10 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      * @return A {@link DetectLanguageResult} equivalent for the error-ed document.
      */
     private DetectLanguageResult convertToErrorDetectLanguageResult(final DocumentError errorDocument) {
-        return new DetectLanguageResult(errorDocument.getId(), null, errorDocument.getError(),
-            null, null);
+        com.azure.cs.textanalytics.implementation.models.Error serviceError = errorDocument.getError();
```
Comment (mssfang) [inline sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java]:
```suggestion
        Error serviceError = errorDocument.getError();
```

File: sdk/textanalytics/azure-cs-textanalytics/src/test/java/com.azure.cs.textanalytics/TextAnalyticsAsyncClientTest.java [hunk 3]
```diff
@@ -127,12 +113,13 @@ public void detectLanguagesNullInput() {
     }
 
     /**
-     * Verifies that the error result is returned when empty text is passed.
+     *  Verifies that an error document is returned for a empty text input.
```
Comment (mssfang) [inline sdk/textanalytics/azure-cs-textanalytics/src/test/java/com.azure.cs.textanalytics/TextAnalyticsAsyncClientTest.java]:
```suggestion
     * Verifies that an error document is returned for a empty text input.
```

Comment (mssfang) [inline sdk/textanalytics/azure-cs-textanalytics/src/test/java/com.azure.cs.textanalytics/TextAnalyticsAsyncClientTest.java]:
```suggestion
     * Verifies that an error document is returned for an empty text input.
```

Comment (mssfang) [inline sdk/textanalytics/azure-cs-textanalytics/src/test/java/com.azure.cs.textanalytics/TextAnalyticsAsyncClientTest.java]:
```suggestion
     * Verifies that an error document is returned for empty text input.
```

**Review by mssfang — APPROVED** [review]
Just a few nit. LGTM

File: sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java (hunk 2 above)
Comment (samvaity) [reply sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java]: As we have two models with the same name, we will have to use this to distinguish them

**Pull request merged**